config/contrib.json

```diff
@@ -6,7 +6,7 @@
     "versions": {
       "storage": {
         "git": {
-          "prefixMessageToSnapshotId": "This version was recorded after filtering snapshot with Mongo ID ",
+          "snapshotURI": "with Mongo ID %SNAPSHOT_ID",
```

Comment:
I don't understand. This is not a [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier), but a text.

![](https://upload.wikimedia.org/wikipedia/commons/thumb/d/d6/URI_syntax_diagram.svg/2136px-URI_syntax_diagram.svg.png)

Reply:
Ok, can you help me to find an appropriate name for this variable? I have trouble finding a name that works well

Reply:
snapshotUniqueIdentifier ?